GamerpotFrontend/src/components/News/styles.ts

```diff
@@ -0,0 +1,35 @@
+import {StyleSheet} from 'react-native';
+import {HEIGHT, WINDOW_WIDTH} from '../../library/constants/carouselConstants';
+import {colors} from '../../library/constants/colors';
+
+const width = WINDOW_WIDTH;
+const height = HEIGHT;
```

Comment:
Why are you doing this assignment instead of using the constants you are importing?